tensorboard/webapp/widgets/data_table/data_table_component.ts

```diff
@@ -32,6 +32,10 @@ import {
   relativeTimeFormatter,
 } from '../line_chart_v2/lib/formatter';
 
+enum Side {
+  RIGHT,
+  LEFT,
+}
```

Comment:
nitpick: would you mind adding a line break here?
```suggestion
}

```

Reply:
Done.